### ISSUE TITLE
Invalid install commandline in postinst

### ISSUE THREAD

**Issue by shevek** [nebula-plugins/gradle-ospackage-plugin#80]
https://github.com/nebula-plugins/gradle-ospackage-plugin/blob/1e39e1102e55c1a313cc4253988df6b8fea6cc79/src/main/groovy/com/netflix/gradle/plugins/deb/DebCopyAction.groovy#L267
Generates "root:root" as the dir's owner.
`man install` says this is invalid, and the group should be passed as `-g`

The actual upshot is this:

```
install -o root:root -d /opt
install: invalid user `root:root'
dpkg: error processing compilerworks-server (--install):
 subprocess installed post-installation script returned error exit status 1
```

**Comment by shevek** [comment]
Presumed fix: Put group into dir.group in DebCopyAction, and generate a more appropriate install line in postinst.ftl ?

**Comment by shevek** [comment]
Seems to stem from DebCopyAction.groovy:
`        String group = lookup(specToLookAt, 'permissionGroup') ?: debTask.permissionGroup`
The bug requires debTask.permissionGroup to be set in order to manifest.

**Comment by bmuschko** [comment]
Would you be interested in providing a pull request for this?

**Comment by luyseyal** [comment]
I'd love to see a fix for this (especially in the 2.2 branch).

**Comment by luyseyal** [comment]
This is still a bug as of 3.2.1 @shevek @bmuschko

**Comment by sghill** [comment]
Fixed with the release of `v4.4.0`
